x-pack/solutions/security/plugins/security_solution/server/ui_settings.ts

```diff
@@ -253,6 +254,23 @@ export const initUiSettings = (
       schema: schema.boolean(),
       solution: 'security',
     },
+    [ENABLE_ASSET_INVENTORY_SETTING]: {
+      name: i18n.translate('xpack.securitySolution.uiSettings.enableAssetInventoryLabel', {
+        defaultMessage: 'Enable Asset Inventory',
```

Comment:
This should be the correct label as per the [Epic](https://github.com/elastic/security-team/issues/11651), apologies for the confusion on the ticket:

```suggestion
        defaultMessage: 'Enable Security Asset Inventory',
```